js/common/lib/inference-session.ts

```diff
@@ -246,6 +246,7 @@ export declare namespace InferenceSession {
     deviceType?: 'cpu'|'gpu'|'npu';
     numThreads?: number;
     powerPreference?: 'default'|'low-power'|'high-performance';
+    contextId?: string;
```

Comment:
It looks like this `contextId` is not expected to be offered by user. Instead, it is set and used internally. Should not add this property to API.

Instead, you can create a new type that extends this and add contextId there in the code implementation.

Reply:
No longer applicable